### DISCUSSION TITLE
Use 301 instead of 302 redirect header

### DISCUSSION THREAD

**Discussion by martijnmichel** [coollabsio/coolify#3751] [Improvement Requests]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Example public repository

coolify

### Description

Using a 302 redirect header for changing between www and non-www is not recommended for seo and prevents us from using coolify in production. ref: [https://ahrefs.com/blog/301-vs-302-redirects/]

### Steps To Reproduce

Create an application with FQDN

### Version

latest

**Comment by Jolg42** [comment]
I had a quick look
I found 2 commits that are related 
https://github.com/coollabsio/coolify/commit/59325406309f21546fa9f036eb41f23bb0d47a67
https://github.com/coollabsio/coolify/commit/aec1d184c8e1fa73ae135fa6d35222ac485789a6

But I can't find anything on the current `main` branch.

Now there is an example for a traefik config here: https://github.com/coollabsio/coolify/blob/main/examples/traefik-dynamic-catch-all.yaml

I guess for you you could change this and set `permanent: true` to get a 301.

**Comment by albertorizzi** [comment]
I made a PR for this https://github.com/coollabsio/coolify/pull/11344
